docs/design/autoware-interfaces/.pages

```diff
@@ -0,0 +1,4 @@
+nav:
+  - index.md
+  - AD API: ad-api
+  - Components: components
```

Comment:
[Nits] Unnecessary?

```suggestion
  - components
```

Reply:
Ah, that's right. I confused what is used for menu text. Directory name is used for directory, and page title for markdown.

Reply:
https://github.com/autowarefoundation/autoware-documentation/pull/57/commits/3b1dade01f4c3d124ed704f949589c99222d13df